gallery/visualtests/nxTag.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { clickTest, focusTest, focusAndHoverTest, hoverTest, simpleTest } = require('./testUtils');
+const { Region, Target } = require('@applitools/eyes-webdriverio');
+
+describe('NxTag', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxTag');
+  });
+
+  describe('Basic NxTag', function() {
+    const selector = '#nx-tag-example .gallery-example-live';
+
+    it('looks right', simpleTest(selector));
+  });
+
+  describe('NxTag with overflow has a tooltip', function() {
+    const selector = '.gallery-example-live',
+          tagSelector = `${selector} .nx-tag--pink`,
+
+          // expected distance from top of element to the top of its tooltip
+          tooltipHeightOffset = 45;
+
+    it('looks right', async function() {
```

Comment:
I'd say this should be in the "Basic NxTag" block above, with the test itself renamed to something like `it('has an overflow tooltip'`

Reply:
> I'd say this should be in the "Basic NxTag" block above, with the test itself renamed to something like `it('has an overflow tooltip'`

OK: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/314856f8ce6cd42ccc42061aff59e0168c9ed38b